### ISSUE TITLE
Failure when writing to Cosmos MongoDB Service

### ISSUE THREAD

**Issue by dgomezc** [Microsoft/WebTemplateStudio#1717]
**Describe the bug**
When writing to a Cosmos MongoDB service from a generated app we get the following error:
"Retryable writes are not supported. Please disable retryable writes by specifying "retrywrites=false" in the connection string or an equivalent driver specific config." 

**To Reproduce**

1. Launch Web Templates Studio
1. Select any frontend and backend framework 
1. Add a List page and a Cosmos DB Service with MongoDB
1. Click create
1. Accept connection string replacement dialog
1. Start the app
1. Try to add something to the list
1. See error

**Comment by dgomezc** [comment]
To solve this, add the property `retryWrites = false` in your backend framework: 

- Node/Express:
  - Open backend/mongo/mongoConnect.js file.
  - In `connect` function, add `useUnifiedTopology: true` and `retryWrites: false` below `useNewUrlParser: true`.

```js
function connect() {
  mongoose
    .connect(`${process.env.COSMOSDB_CONNSTR}?ssl=true&replicaSet=globaldb`, {
      auth: {
        user: process.env.COSMOSDB_USER,
        password: process.env.COSMOSDB_PASSWORD
      },
      useNewUrlParser: true,
      useUnifiedTopology: true,
      retryWrites: false
    })
    .then(() => console.log("Connection to CosmosDB successful"))
    .catch(err => console.error(err));
}
```

- Flask:
  - Open backend/src/mongo/client.py file.
  - Add `retryWrites=false` to connection string in `MongoClient()`.
```python
client = MongoClient(connection_str + '?ssl=true&replicaSet=globaldb&retryWrites=false')
```

- ASP.NET
  - Open backend/startup.cs file.
  - In `InitializeCosmosClientInstance()` method add `settings.RetryWrites = false;` before `return new MongoClient(settings);`.

 ```csharp
private IMongoClient InitializeCosmosClientInstance()
{
    var connectionString = Configuration["COSMOSDB_CONNSTR"];
    var userName = Configuration["COSMOSDB_USER"];
    var dbName = Configuration["COSMOSDB_DB_NAME"];
    var password = Configuration["COSMOSDB_PASSWORD"];
    
    var settings = MongoClientSettings.FromConnectionString(connectionString);
    
    var identity = new MongoInternalIdentity(dbName, userName);
    var evidence = new PasswordEvidence(password);
    settings.Credential = new MongoCredential("SCRAM-SHA-1", identity, evidence);
    
    settings.UseTls = true;
    settings.SslSettings = new SslSettings() { EnabledSslProtocols = SslProtocols.Tls12 };
    settings.RetryWrites = false;
    
    return new MongoClient(settings);
}
```

- Moleculer
  - Open backend/mixins/db.mixin.js file.
  - In MongoAdapter constructor, add `retryWrites: false` below `useUnifiedTopology: true`.

```js
adapter: new MongoAdapter(
  `${process.env.COSMOSDB_CONNSTR}?ssl=true&replicaSet=globaldb`,
  {
    auth: {
      user: process.env.COSMOSDB_USER,
      password: process.env.COSMOSDB_PASSWORD
    },
    useNewUrlParser: true,
    useUnifiedTopology: true,
retryWrites: false
  }
),
```

**Comment by prupipho** [comment]
Fixed and tested in version: 0.5.2104801
